src/lib.rs

```diff
@@ -168,32 +487,124 @@ impl Child {
         }
     }
 
-    pub fn kill(&self) -> Result<()> {
+    /// Returns process identifier associated with this child.
+    ///
+    /// Equivalent to the [`GetProcessId`][get-process-id] function.
+    ///
+    /// # Examples
+    ///
+    /// ```no_run
+    /// use CreateProcessW::Command;
+    ///
+    /// let mut command = Command::new("notepad.exe");
+    ///
+    /// if let Ok(child) = command.spawn() {
+    ///     match child.id() {
+    ///         Ok(id) => println!("Child's ID is {}", id),
+    ///         Err(err) => println!("Cannot get child's ID"),
+    ///     }
+    /// } else {
+    ///     println!("notepad didn't start");
+    /// }
+    /// ```
+    ///
+    /// [get-process-id]: https://docs.microsoft.com/en-us/windows/win32/api/processthreadsapi/nf-processthreadsapi-getprocessid
+    pub fn id(&self) -> Result<u32> {
```

Comment:
This does not reflect std's API properly because of the result.

I think you can use the special value `0` for an error here.